app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+class PostsController < ApplicationController
+  def index
+    @posts = Post.all
+  end
+
+  def new
+    @post = Post.new
+  end
+
+  def create
+    @user = User.find_by(id: current_user.id)
```

Comment:
**`OPTIONAL`** I suggest you define a `current_user` from `ApplicationController` so that you will use it everywhere in your code